transaction.go

```diff
@@ -0,0 +1,57 @@
+package driver
+
+// TransactionOptions contains options that customize the transaction.
+type TransactionOptions struct {
+	// Transaction size limit in bytes. Honored by the RocksDB storage engine only.
+	MaxTransactionSize int
+
+	// An optional numeric value that can be used to set a timeout for waiting on collection
+	// locks. If not specified, a default value will be used.
+	// Setting lockTimeout to 0 will make ArangoDB not time out waiting for a lock.
+	LockTimeout interface{}
```

Comment:
Why `interface{}`? 
Probably you want to use and `*int`

Reply:
I wanted to be able to write `TransactionOptions{LockTimeout: 5}` instead of

    lockTimeout := 5
    TransactionOptions{LockTimeout: &lockTimeout}

but if type-safety is not a valid trade-off at this point, I can change that.

Reply:
I understand the argument, but I prefer type safety over ease of use in this case.